docs/whats-new.asciidoc

```diff
@@ -1,189 +1,56 @@
-* 33 new <<prebuilt-rules, prebuilt rules>>.
-* A new <<cases-ui-integrations, Swimlane connector for cases and alerts>> allows you to
-forward alerts and cases to your Swimlane application for enhanced incident and workflow management.
-* Admins can now specifically assign `All`, `Read`, or `None` <<case-permissions, privileges>> to users for the Cases feature.
-* New functionality and usability improvements were added to the https://github.com/elastic/integrations/tree/master/packages/osquery_manager[Osquery Manager integration], including the ability to save and edit a library of queries, constraining queries to specific operating systems, and more.
-* <<event-filters, Event filters>> allow admins to prevent endpoint events they do not need or want stored from streaming - allowing better storage utilization in {es}.
+Other versions: {security-guide-all}/7.17/whats-new.html[7.17]| {security-guide-all}/7.16/whats-new.html[7.16] | {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
+{security-guide-all}/7.9/whats-new.html[7.9]
 
 [discrete]
-[[sec-experimental-7.14]]
-=== Experimental features
-
-IMPORTANT: The following features are experimental and may be changed or removed completely in a future release. Elastic will take a best effort approach to fix any issues, but experimental features are not subject to the support of official GA features.
+[[name-changes-8.0]]
+== Terminology changes
 
-* Host Risk Score: A framework to figure out the most suspicious hosts in your environment, based on their alert activity.
-For more information, click https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md[here].
-* A new experimental https://github.com/elastic/detection-rules/tree/main/docs/experimental-machine-learning[URL Spoofing] model framework allows you to proactively detect and monitor spoofing activity by raising an alert whenever you interact with a predicted malicious URL in your environment. Requires Platinum or higher https://www.elastic.co/subscriptions[subscription].
+*Index patterns* has been renamed to *Data views*
 
+A {kibana-ref}/data-views.html[data view] data view selects the data to use and allows you to define properties of the fields. A data view can point to one or more indices, data stream, or index aliases. Please note that *index patterns* only refers to a group of defined index patterns. In {elastic-sec}, the default security data view includes index patterns from `auditbeat-*`, `filebeat-*`, `logs-*`, and `winlogbeat-*`. You can {kibana-ref}/data-views.html#settings-create-pattern[create a new data view] to update these settings.
```

Comment:
I'm not certain if "data streams" should be plural, but if not I'd change to "_a_ data stream".

Also, is it worth noting that data views are a Kibana-wide change, not just Elastic Security?

```suggestion
A {kibana-ref}/data-views.html[data view] selects the data to use and allows you to define properties of the fields. A data view can point to one or more indices, data streams, or index aliases. Please note that *index patterns* only refers to a group of defined index patterns. In {elastic-sec}, the default security data view includes index patterns from `auditbeat-*`, `filebeat-*`, `logs-*`, and `winlogbeat-*`. You can {kibana-ref}/data-views.html#settings-create-pattern[create a new data view] to update these settings.
```